Topics/User_Experience/navigation design in complex websites.md

```diff
@@ -0,0 +1,64 @@
+## What is Navigation Design ?
+Navigation Design, also referred to as menu design, helps users to easily move through the pages of your website. It utilizes design techniques to allow ease of navigation and provide the user a better sure experience. 
+
+
+## Good vs Bad Navigation Design 
+Good navigation design allows for ease of  finding information, is intuitive and easy to use. On the other hand, bad navigation design can overwhelm the user and make finding important content difficult.  As a result,  it is important to keep the user in mind when designing the navigation of a website, especially complex websites with a multitude of pages. Reasoning about how the user can navigate through the many pages of a website, can help lay the foundation to good navigation design. 
+
+Consider using the following steps to start thinking of good navigation design  : 
+
+1. Layout all the pages on your website whether on paper of through a design tool such as Figma 
+2.  Find the relevant ways a user may navigate through those pages 
+    1. For example, home page to sign in back to home page 
+3. Determine which pages they user may need to reference most often
+    2. For example, the home page / landing page, 
+4. Consider using dome of the following to ease navigation through the website :  
+    * Navbars
+        * Usually at the top of a landing page / home page. These contain links to the website's main pages. 
+    * Sub-navbars
+        * Located below a normal Navbar and  consists of links that are relevant to the specific page that the user is on 
+    * Side menus
+        * Usually located at the side of the page and contains links to relevant pages on the website 
+    * Tabbed menus
+        * Organizes information into different tabs, where each tab contains different relevant information
+    * Footers containing links to all relevant pages within your website 
+        * Usually at the bottom of a page. It can contain links to all pages on the website as well as contact information.
+          
+* Please see the link below for more navigation design patterns:
+    * [https://ui-patterns.com/patterns/navigation/list](https://ui-patterns.com/patterns/navigation/list)
+      
+
+## Advantages of Good Navigation Design for Complex Websites  
+
+* Good navigation design allows a clear or  intuitive path for users to navigate through the web app / website 
```

Comment:
I think this point (line 33) and the point below (line 39) can be combined into one. 

> An easy/ clear path through a website to specific content, can be an advantage in encouraging users to use your product

Reply:
Great idea ! Reviewing this further, it does seem a bit repetitive.